core/src/main/java/org/infinispan/factories/threads/BlockingThreadFactory.java

```diff
@@ -3,9 +3,11 @@
 import org.infinispan.commons.executors.BlockingResource;
 
 public class BlockingThreadFactory extends DefaultThreadFactory implements BlockingResource {
-   public BlockingThreadFactory(String threadGroupName, int initialPriority, String threadNamePattern,
-         String node, String component) {
-      super(new ISPNBlockingThreadGroup(threadGroupName), initialPriority, threadNamePattern, node, component);
+
+   private static final ISPNBlockingThreadGroup DEFAULT_THREAD_GROUP = new ISPNBlockingThreadGroup("ISPN-blocking-thread-group");
```

Comment:
Same here just use the blocking one at We don't need this one, just use the one at DefaultThreadFactory#BlockingThreadGroupHolder and make it public.